hack/docs/compatibilitymatrix_gen/compatibility.yaml

```diff
@@ -53,4 +53,7 @@ compatibility:
     maxK8sVersion: 1.30
   - appVersion: 1.0.x
     minK8sVersion: 1.25
-    maxK8sVersion: 1.30
\ No newline at end of file
+    maxK8sVersion: 1.30
+  - appVersion: 1.0.5
```

Comment:
I had never actually seen how this package works with a patch version -- interesting

Reply:
It doesn't add the greater than or equal until an overlap is added like when we do `1.1.x`. I've manually added it for now.